views/Home.jsx

```diff
@@ -1,13 +1,21 @@
 const React = require('react');
-
+const Layout = require('./layout')
 function Home() {
 
   return (
-    <main>
-      <h1>Home</h1>
-    </main>
+    <Layout>
+      <main>
+        <h1>Home</h1>
+        <img src=".././public/images/beer.png" alt="image of beer" />
```

Comment:
The image is not linked correctly. From now on, you don't need to refer to the public folder. 

Just write:
`<img src="../images/beer.png" alt="image of beer" />`